src/Swarm/Game/Robot.hs

```diff
@@ -431,6 +433,8 @@ tickSteps :: Lens' Robot Int
 
 -- | Is the robot currently running an atomic block?
 runningAtomic :: Lens' Robot Bool
+emptyRobotContext :: RobotContext
```

Comment:
Can we put a blank line in between `runningAtomic` and `emptyRobotContext`?  A Haddock comment might be nice too, though I guess `emptyRobotContext` is pretty self-explanatory.

Reply:
Also, is there a particular reason for putting `emptyRobotContext` here?  I would tend to put it near the definition of `RobotContext`.